features/lessons/src/main/java/com/cryart/sabbathschool/lessons/ui/quarterlies/SSQuarterlyItemViewModel.kt

```diff
@@ -89,6 +89,7 @@ internal class SSQuarterlyItemViewModel(
         fun loadCover(view: ImageView, coverUrl: String?, primaryColor: Int?) {
             coverUrl?.let { url ->
                 view.load(url) {
+                    allowHardware(false)
```

Comment:
Fixes crash on api 26 devices

Reply:
It's not fear to disable hardware acceleration for all devices. Maybe just disable crossfade for API 26?

Reply:
Updated